lib/active_model_serializers/rspec/matchers/serializer.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_model_serializers/test/serializer'
+
+module ActiveModelSerializers
+  module RSpecMatchers
+    module Serializer
+      extend ActiveModelSerializers::Test::Serializer
+      extend ActiveSupport::Concern
+
+      included do
+        setup :setup_serialization_subscriptions
+        teardown :teardown_serialization_subscriptions
+
+        RSpec::Matchers.define :use_serializer do |expected|
```

Comment:
It's the subscriber to the`::ActiveModelSerializers::Logging::RENDER_EVENT` that allows us to see what serializer was used. At least that's how the current assertion works. That is, of course, if we want to keep this assertion/matcher.  See https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model_serializers/test/serializer.rb#L67